python/tchannel/tchannel.py

```diff
@@ -20,13 +21,14 @@
 class TChannel(object):
     """Manages inbound and outbound connections to various hosts."""
 
-    def __init__(self, process_name=None):
+    def __init__(self, process_name=None, in_port=None):
         self.peers = {}
         self.process_name = (
             process_name or "%s[%s]" % (sys.argv[0], os.getpid())
         )
 
-        # TODO: self.server_socket = self.make_in_connection()
+        self.inbound_server = InboundServer()
+        self.make_in_connection(in_port)
```

Comment:
Can we consolidate the inbound/outbound stuff is some `Connection` class methods, e.g. `Connection.inbound(listen_port)` and `Connection.outbound(host, port)`? The single-threaded case wouldn't implement `inbound`.